moto/sns/models.py

```diff
@@ -193,6 +195,9 @@ def update_sms_attributes(self, attrs):
         self.sms_attributes.update(attrs)
 
     def create_topic(self, name):
+        fails_constraints = not re.match(r'^[a-zA-Z0-9](?:[A-Za-z0-9_-]{0,253}[a-zA-Z0-9])?$', name)
+        if fails_constraints:
+            raise InvalidParameterValue("Can only include alphanumeric characters, hyphens, or underscores. 1 to 80 in length")
```

Comment:
1 to 256 ?

Reply:
Just copy and pasted what you gave me :)

Reply:
I meant the string in the exception